Plugins/Gerrit/FormGerritBase.cs

```diff
@@ -66,7 +66,17 @@ private Version GetGerritVersion()
         {
             RestRequest request = new RestRequest("/config/server/version");
             IRestResponse response = client.Execute(request);
-            return Version.Parse(response.Content.Replace(")]}'", "").Replace("\n", "").Replace("\"", "").Split('-')[0]);
+
+            return (response.IsSuccessful
+                && Version.TryParse(ExtractVersion(response.Content), out var version))
+                ? version : GerritUtil.GerritWithPrivateReviewSupport;
```

Comment:
We still need to wrap it in `try..catch`, otherwise we risk crashing the app again, if something is wrong.

Reply:
Looking at possible error sources:
 - rest request handles exceptions - see https://github.com/restsharp/RestSharp/blob/master/RestSharp/RestClient.Sync.cs
 - Version.TryParse does not throw
 - Regex.Match might throw if input is null or regex is invalid.

 However in this last case, as long as we intend to handle it, should we just log it or just show a message box? (need some guidance on the error handling mechanism on GitEx)